EarlGrey/Action/GREYActions.m

```diff
@@ -330,6 +330,12 @@ + (void)grey_webSetText:(id)element text:(NSString *)text {
       [GREYActions grey_webSetText:element text:text];
     } else {
       [element setText:text];
+      [element sendActionsForControlEvents:UIControlEventEditingChanged];
```

Comment:
@Arcank do we need to also do this for setting the text directly on web elements?